src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -18,7 +18,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ImportGroup>
 
   <PropertyGroup>
-    <DefaultExcludes>bin\**;obj\**</DefaultExcludes>
+    <DefaultExcludes>bin\**;obj\**;packages\**</DefaultExcludes>
```

Comment:
I see, so this does not get pulled into the excludes for compile and embedded resources? That's where we have this exclude in the CLI.